src/common/routes.ts

```diff
@@ -29,6 +29,7 @@ export default {
     EDIT: `/:username(@[\\w\\.\\d-]+)/:permlink/edit`,
     EDIT_DRAFT: `/draft/:draftId`,
     DISCOVER: `/discover`,
+    SEARCH_MAIN: `/search-main/`,
```

Comment:
Why not just search?

Reply:
changed routing structure.

`/search` is main search page.

`/search-more` allows scrolling into search results.

Reply:
`/search-main` you mean right?

Reply:
removed `/search-main` created `/search-more` instead. sounds like more appropriate :)